src/components/form/AddNewBook.js

```diff
@@ -3,6 +3,7 @@ import { v4 as uuidv4 } from 'uuid';
 import { useDispatch } from 'react-redux';
 import { postBook } from '../../redux/books/books';
 
+// Add New Book Form Component
 function AddNewBook() {
```

Comment:
Please make sure the app has the same functionality of adding and removing books and showing all the books even after refactoring.